{
  "schema_version": "1.6.3",
  "id": "CVE-2024-6126",
  "details": "A flaw was found in the cockpit package. This flaw allows an authenticated user to kill any process when enabling the pam_env's user_readenv option, which leads to a denial of service (DoS) attack.",
  "aliases": [],
  "related": [],
  "published": "2024-07-03T15:15:00Z",
  "modified": "2024-07-03T15:15:00Z",
  "affected": [
    {
      "package": {
        "ecosystem": "Ubuntu:Pro:18.04:LTS",
        "name": "cockpit"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            }
          ]
        }
      ],
      "versions": [
        "151-1",
        "156-1",
        "157-1",
        "158-1",
        "160-1",
        "161-1",
        "162-1",
        "163-1",
        "164-1"
      ]
    },
    {
      "package": {
        "ecosystem": "Ubuntu:20.04:LTS",
        "name": "cockpit"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            }
          ]
        }
      ],
      "versions": [
        "202.1-1",
        "204-1",
        "206-1",
        "207-1",
        "208-1",
        "210-1",
        "211-1",
        "212-1",
        "213-1",
        "214.1-1",
        "215-1"
      ]
    },
    {
      "package": {
        "ecosystem": "Ubuntu:22.04:LTS",
        "name": "cockpit"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            }
          ]
        }
      ],
      "versions": [
        "252-1",
        "256-1",
        "257-1",
        "258-1",
        "259-1",
        "260-1",
        "261-1",
        "262-1",
        "263-1",
        "264-1",
        "264-1ubuntu0.22.04.1"
      ]
    },
    {
      "package": {
        "ecosystem": "Ubuntu:24.04:LTS",
        "name": "cockpit"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            }
          ]
        }
      ],
      "versions": [
        "300.1-1",
        "303-1",
        "304-1",
        "305-1",
        "306-1",
        "307-1",
        "308-1",
        "309-1",
        "310.1-1",
        "311-1",
        "312-1build2",
        "314-1"
      ]
    }
  ],
  "references": [
    {
      "type": "REPORT",
      "url": "https://ubuntu.com/security/CVE-2024-6126"
    },
    {
      "type": "REPORT",
      "url": "https://www.cve.org/CVERecord?id=CVE-2024-6126"
    },
    {
      "type": "REPORT",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2292897"
    },
    {
      "type": "REPORT",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2290859"
    },
    {
      "type": "REPORT",
      "url": "https://access.redhat.com/security/cve/CVE-2024-6126"
    }
  ]
}